src/OrchardCore/OrchardCore.ContentManagement/Handlers/ContentPartHandlerCoordinator.cs

```diff
@@ -223,6 +115,8 @@ public override async Task LoadingAsync(LoadContentContext context)
             foreach (var typePartDefinition in contentTypeDefinition.Parts)
             {
                 var partName = typePartDefinition.PartDefinition.Name;
+                if (string.IsNullOrEmpty(partName))
+                    throw new System.ArgumentNullException("partName cannot be null for contentType:" + contentTypeDefinition.Name);
```

Comment:
Instead of throwing an exception, log it and continue "so it skips the execution" instead.